docs/topics/mta-rn-resolved-issues-6-2-0.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-6.2/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-6-2-0_{context}"]
+= Resolved issues
+
+The following highlighted issues have been resolved in MTA version 6.2.0: 
+
+.Analysis wizard
+The release of MTA 6.2.0 resolves the issue that Analysis wizard was stuck on custom rules page on moving *Back* from the Repository tab.
+For more information on this issue, see link:https://issues.redhat.com/browse/MTA-464[MTA-464].
+
+.Tags & Reports tabs
+The release of MTA 6.2.0 resolves the issue that an analysis was running for an application, and after clicking on that application to see the Tags and Reports, both the tabs keep loading until the analysis finished.
+For more information on this issue, see link:https://issues.redhat.com/browse/MTA-465[MTA-465].
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12418203[Resolved Issues in Jira].
```

Comment:
Thanks @HagayVider1 looks better. I think you need to tighten up the language and format it better.
But it works well as a draft.